src/realtime/flood/flood_hazard.py

```diff
@@ -181,8 +184,7 @@ def write_keywords(self):
 
             # Extra keywords
             property_extra_keywords['key']: {
-                extra_keyword_flood_event_time['key']: self.event_time.strftime(
```

Comment:
hm so it was there already, but why we don't get the `flood_event_time` before this PR ? strange..

Reply:
Haha no, it was not there before. This line is just partial diff of the PR.